### ISSUE TITLE
Missing "jest" dependency?

### ISSUE THREAD

**Issue by tobilg** [mhlabs/cfn-diagram#8]
I just installed the latest version as outline in the README. I get the following error when trying to run the tool:

```bash
$ cfn-dia
internal/modules/cjs/loader.js:985
  throw err;
  ^

Error: Cannot find module 'jest'
Require stack:
- /home/user/.nvm/versions/node/v12.16.1/lib/node_modules/@mhlabs/cfn-diagram/graph/Vis.js
- /home/user/.nvm/versions/node/v12.16.1/lib/node_modules/@mhlabs/cfn-diagram/index.js    
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/home/user/.nvm/versions/node/v12.16.1/lib/node_modules/@mhlabs/cfn-diagram/graph/Vis.js:5:27)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/user/.nvm/versions/node/v12.16.1/lib/node_modules/@mhlabs/cfn-diagram/graph/Vis.js',
    '/home/user/.nvm/versions/node/v12.16.1/lib/node_modules/@mhlabs/cfn-diagram/index.js'
  ]
}
```

I'm unsure if this is a problem with my personal setup, or a missing (peer) dependency of the project.

**Comment by ljacobsson** [comment]
Hi @tobilg,

Thanks for reporting. This looks identical to #7 and should be solved since v1.1.1. I just spun up a fresh ec2 instance and tried to install it and it works as expected.

Can you please verify which version you're on `cfn-dia -v`?

Latest version is 1.1.2

**Comment by tobilg** [comment]
Thanks for the fast feedback. Apparently this was caused by a version 1.1.0 install in the `/home/user/.nvm/versions/node/v12.16.1/lib/node_modules/@mhlabs/cfn-diagram` folder, which wasn't unistalled by npm for whatever reason possible. 

After deleting the folder and a fresh install, it now work. Closing this issue now. Thanks!
